tools/benchmark.py

```diff
@@ -120,6 +120,8 @@ def run_syscall_count_benchmark(deno_path):
     syscall_count_map = {}
     syscall_count_map["hello"] = get_strace_summary(
         [deno_path, "tests/002_hello.ts", "--reload"])["total"]["calls"]
+    syscall_count_map["fetch_deps"] = get_strace_summary(
+        [deno_path, "tests/fetch_deps.ts", "--reload"])["total"]["calls"]
```

Comment:
I believe this requires `allow-net` flag, Travis is failing and complains “PermissionDenied”